azero/contracts/most/lib.rs

```diff
@@ -898,6 +972,16 @@ pub mod most {
             Ok(())
         }
 
+        /// Set wazero(azero) psp22 token contract
+        ///
+        /// Can only be called by the contracts owner
+        #[ink(message)]
+        pub fn set_wazero(&mut self, wazero_address: AccountId) -> Result<(), MostError> {
+            self.ensure_owner()?;
```

Comment:
Question: shouldn't this have `ensure_halted` too? We will most likely only ever call this once, but still?

Reply:
Probably should